app/scripts/abotypen/detail/abotypendetail.html

```diff
@@ -182,8 +182,8 @@ <h3 class="panel-title" translate>Zusatzdaten</h3>
                 <div class="input-group">
                   <div class="input-group-addon" ng-if="getModel() === 'abotypen'" >{{projekt.waehrung}}</div>
                   <div class="input-group-addon" ng-if="getModel() === 'zusatzAbotypen'" >{{projekt.waehrung}}</div>
-                  <input type="number" ng-if="getModel() === 'abotypen'" placeholder="{{'Preis'|translate}}" ng-pattern="/^[0-9]+([\.,][0-9]{1,3})?$/" step="0.01" class="form-control" name="zielpreis" ng-model="abotyp.zielpreis"></input>
-                  <input type="number" ng-if="getModel() === 'zusatzAbotypen'" placeholder="{{'Preis'|translate}}" ng-pattern="/^[0-9]+([\.,][0-9]{1,3})?$/" step="0.01" class="form-control" name="zielpreis" ng-model="zusatzAbotyp.zielpreis"></input>
+                  <input type="number" ng-if="getModel() === 'abotypen'" placeholder="{{'Preis'|translate}}" ng-pattern="/^[0-9]+([\.,][0-9]{1,3})?$/" step="0.01" min="0" max="99999.99" class="form-control" name="zielpreis" ng-model="abotyp.zielpreis"></input>
+                  <input type="number" ng-if="getModel() === 'zusatzAbotypen'" placeholder="{{'Preis'|translate}}" ng-pattern="/^[0-9]+([\.,][0-9]{1,3})?$/" step="0.01" min="0" max="99999.99" class="form-control" name="zielpreis" ng-model="zusatzAbotyp.zielpreis"></input>
```

Comment:
This has not been the case so far, but can we imagine a Abo or Zusatzabo with a negative price? This could be the case if someone is getting a discount of some sort?

Reply:
That is right, that was an option until know. I do not particularly like to have this "hack" to apply some "discount" or "voucher". I would be more in favor of implementing this functionality in case someone needs it. However, this would represent an effort that I don't think we would want to make in the short term... in any case we would need to define a min to avoid errors in the database, being the biggest min -99999.99